src/BloomExe/Edit/EditingModel.cs

```diff
@@ -535,6 +519,14 @@ private void OnPageSelectionChanging(object sender, EventArgs eventArgs)
 			}
 		}
 
+		public void ShowAddPageDialog()
```

Comment:
This feels as though it should be in Javascript. But I gather you made a C# button? @hatton, is it worth waiting to change this?